config/config.go

```diff
@@ -41,6 +41,7 @@ type AmiConfiguration struct {
 	KmsKeyAliasName    string            `json:"kms_key_alias_name"`
 	Visibility         string            `json:"visibility"`
 	Tags               map[string]string `json:"tags,omitempty"`
+	SharedWithAccounts []string          `json:"shared_with_accounts"`
```

Comment:
Consider `SharedWithUserIds` instead of `...Accounts` to match [the use below](https://github.com/cloudfoundry/bosh-aws-light-stemcell-builder/pull/32/files#diff-1d405178856c6061594269cf1e43590ffab72f1cc2c655b74d6cc15d13eeaa65R121) assigning `ec2.LaunchPermission.UserId`.

Alternate names might be `UserIdsAllowedToLaunch`, or `EnableLaunchForUserIds`, or `PermittedUserIds`, or `LaunchAuthorisedUserIds`, or `NamingIsHardAddUserIdsHereIfTheyHaveAccessOrSomething`....

Reply:
Hmm I do understand your concerns, but I did not find an example where the UserId isn't an account and `AddLaunchPermissionsForAccounts` is just to long. On the other hand it is at least similar to the AWS Web Console wording "Shared accounts". Therefor I'll keep it.